frontend/website/jobs/visual-motion-designer.markdown

```diff
@@ -0,0 +1,52 @@
+---
+title: Visual & Motion Designer
+date: 2019-11-06
+---
+
+O(1) Labs is seeking a Visual & Motion Designer who can navigate a variety of mediums to deliver world-class visuals, illustrations, animations, and video content for Coda Protocol.
+
+In the nascent sector of crypto design, trustworthy and delightful designed experiences are largely absent. We aim to create engaging visuals and video content that are easily understood and enjoyed by an international audience.
+
+Visual Designers will work closely with Marketing to develop graphics, videos and campaigns in a variety of mediums.
```

Comment:
if we capitalize marketing, should also capitalize product, engineering to be consistent